deployment/wsgi/gunicorn.conf.py

```diff
@@ -0,0 +1,7 @@
+wsgi_app = "deployment.wsgi.wsgi:application"
+bind = "0.0.0.0:8000"
+workers = 1
```

Comment:
how are we deciding whether or not to set this to 1? I'm all for the explicit-ness (even though I believe the default is 1), but I'm wondering if there was some testing with more that failed so we fell back to this? I see other services with more workers like indexd